src/components/attendee/BasicInfo/ProfileModalAvatar/ProfileModalAvatar.module.scss

```
@@ -1,22 +1,30 @@
+@use "scss/attendee/space";
+@use "scss/attendee/theme";
+@use "scss/attendee/font";
+@use "scss/attendee/border";
+
 .profileModalAvatar {
-  width: 84px;
-  height: 84px;
+  width: space.empty(7);
+  height: space.empty(7);
 }
 
 .input {
   display: none;
 }
 
 .image {
-  width: 84px;
-  height: 84px;
+  width: space.empty(7);
+  height: space.empty(7);
   border-radius: 50%;
   cursor: pointer;
 }
 
 .text {
-  font-size: 14px;
-  margin: -58px 0px 50px 0px;
+  @include font.core;
+  position: sticky;
+  margin: -(space.empty(5)) space.empty(1) 0 space.empty(1);
```

The reason to make `space.empty()` a function instead of a constant was to allow for better context and for clearer code:

```suggestion
  margin: space.empty(-5) space.empty(1) 0 space.empty(1);
```

---

[a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)